applications/nrf_desktop/README.rst

```diff
@@ -955,6 +957,98 @@ To use the nRF Desktop application with your custom board:
    Some options have dependencies that might not be needed when these options are disabled.
    For example, when the LEDs module is disabled, the PWM driver is not needed.
 
+Adding a new motion sensor
+==========================
+
+The nRF Desktop application comes with a :ref:`nrf_desktop_motion` able to read data from a motion sensor.
+In the scope of NCS two motion sensor drivers are available, for PMW3360 and PAW3212.
+Complete the following steps to add a new motion sensor into the project.
+
+1. Add a new sensor driver
+
+   The first step is to create a new motion sensor driver, that will provide code communicating with the sensor.
+   A communication between the application and the sensor is done through a sensor driver API (see :ref:`sensor_api`).
+   For motion module to work correctly, the driver must support a trigger (see ``sensor_trigger_set``) on a new data (see ``SENSOR_TRIG_DATA_READY`` trigger type).
+   When motion data is ready, the driver calls a register callback.
```

Comment:
Only the first sample after inactivity is fetched this way, then samples are fetched by the thread without using the trigger.

Reply:
Thanks, I have almost forgot ( ͡° ͜ʖ ͡°)

Reply:
Seriously though I don't want to repeat the pipeline process for I think third time. The focus here is on how upper layer interacts with the driver.

Reply:
I just meant that this suggests the reader that sensor reads every sample using the callback (this could be a bit misleading). But we can leave it as it is.